src/tooltip.js

```diff
@@ -194,6 +194,11 @@ angular.module('ts.utils')
             case 'click':
               $element.on('click', toggleVisibility);
               break;
+            case 'hybrid':
+              $element.on('mouseenter', makeVisible);
+              $element.on('mouseleave', makeInvisible);
+              $element.on('click', toggleVisibility);
```

Comment:
In hybrid mode, clicking the element that this directive is attached to will dismiss the tooltip, which will avoid issues like this:
![this](https://s3.amazonaws.com/prod.tracker2/resource/71035421/Pasted%20Image%20at%202016_10_28_11_19%20am.png?response-content-disposition=inline&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAJEX3ET63U5T77TYA%2F20161107%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20161107T182109Z&X-Amz-Expires=1800&X-Amz-SignedHeaders=host&X-Amz-Signature=968169280d71a8ec5780b02a4d7184e3d7104baf1ca5b816a7da759268886582)